man/as.matrix.Rd

```diff
@@ -17,6 +17,11 @@ the \code{rownames} in the returned \code{matrix}. If \code{TRUE} the
 single column, otherwise the first column in the \code{data.table} will
 be used. Alternative a vector of length \code{nrow(x)} to assign as the
 row names of the returned \code{matrix}.}
+\item{rownames.literal}{optional, when x is a \code{data.table} that has
```

Comment:
before word "optional" I would put "logical" to clearly state argument type in front, so `{logical, optional, when ...`.

`x` alone should be wrapped in `\code{x}`
```
... when x is ...
```
to
```
... when \code{x} is ...
```